package.json

```diff
@@ -43,6 +43,7 @@
     "typescript": "3.8.3"
   },
   "dependencies": {
+    "@types/express": "4.17.3",
```

Comment:
it's better as dev dependecy

Reply:
@pastahito actually, I'm not sure. if people write plugins they need the type

Reply:
yes, the aim was to expose interface `PluginRouteOptions` to the developer of the plugin